specification/web/resource-manager/Microsoft.CertificateRegistration/stable/2024-11-01/AppServiceCertificateOrders.json

```diff
@@ -0,0 +1,1801 @@
+      "required": [
+        "value"
+      ],
+      "type": "object",
+      "properties": {
+        "value": {
+          "description": "Collection of resources.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/AppServiceCertificateOrder"
+          }
+        },
+        "nextLink": {
+          "description": "Link to next page of resources.",
+          "type": "string",
+          "readOnly": true
+        }
+      }
+    },
+    "AppServiceCertificateOrderPatchResource": {
+      "description": "ARM resource for a certificate order that is purchased through Azure.",
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "../../../Microsoft.Web/stable/2024-11-01/CommonDefinitions.json#/definitions/ProxyOnlyResource"
+        }
+      ],
+      "properties": {
+        "properties": {
+          "description": "AppServiceCertificateOrderPatchResource resource specific properties",
+          "required": [
+            "productType"
+          ],
+          "type": "object",
+          "properties": {
+            "certificates": {
+              "description": "State of the Key Vault secret.",
+              "type": "object",
+              "additionalProperties": {
+                "$ref": "#/definitions/AppServiceCertificate"
+              }
+            },
+            "distinguishedName": {
+              "description": "Certificate distinguished name.",
+              "type": "string",
+              "x-ms-mutability": [
+                "create",
+                "read"
+              ]
+            },
+            "domainVerificationToken": {
+              "description": "Domain verification token.",
+              "type": "string",
+              "readOnly": true
+            },
+            "validityInYears": {
+              "format": "int32",
+              "description": "Duration in years (must be 1).",
+              "default": 1,
+              "type": "integer",
+              "x-ms-mutability": [
+                "create",
+                "read"
+              ]
+            },
+            "keySize": {
+              "format": "int32",
+              "description": "Certificate key size.",
+              "default": 2048,
+              "type": "integer",
+              "x-ms-mutability": [
+                "create",
+                "read"
+              ]
+            },
+            "productType": {
+              "description": "Certificate product type.",
+              "enum": [
+                "StandardDomainValidatedSsl",
+                "StandardDomainValidatedWildCardSsl"
+              ],
+              "type": "string",
+              "x-ms-enum": {
+                "name": "CertificateProductType",
+                "modelAsString": false
+              },
+              "x-ms-mutability": [
+                "create",
+                "read"
+              ]
+            },
+            "autoRenew": {
```

Comment:
>autoRenew [](http://example.com/codeflow?start=13&length=9)

Prefer enum to boolean, e.g. renewalPolicy: { Manual, Automatic, Default } or similar.